### ISSUE TITLE
Issue with styling

### ISSUE THREAD

**Issue by thewebartisan7** [TarekRaafat/autoComplete.js#137]
I am trying to change class name of result list and item.

Works fine without changing class name, but when I change class name like below then results item is empty, appear only elements li:

```
            resultsList: {
                render: true,
                container: source => {
                    source.setAttribute("class", "dropdown-menu show");
                },
                destination: document.querySelector("#autoComplete"),
                position: "afterend",
                element: "ul"
            },
            resultItem: {
                content: (data, source) => {
                    source.setAttribute("class", "autoComplete_result dropdown-menu-item");
                },
                element: "li"
            },
```

Also how I can add class "show" to resultsList since I want to display inside dropdown-menu which is visible with class "show".

Thanks

**Comment by TarekRaafat** [comment]
Hello @thewebartisan7,

You can add or change `resultsList` class name as shown in the code below or on codepen [demo](https://codepen.io/tarekraafat/pen/rQopdW?editors=0010)


```js
resultsList: {
    render: true,
    container: source => {
    source.setAttribute("id", "autoComplete_list");
        source.setAttribute("class", "autoComplete_list");
    },
    destination: document.querySelector("#autoComplete"),
    position: "afterend",
    element: "ul"
},
```

Try it and let me know the results.


Cheers! :)

**Comment by TarekRaafat** [comment]
Hello @thewebartisan7,

The issue has been resolved in [v8.0](https://github.com/TarekRaafat/autoComplete.js/releases/tag/v8.0.0) and above please try it and let me know your feedback.

Have a nice day, cheers! :)
